backend/src/crd/pkg/apis/scheduledworkflow/v1beta1/types.go

```diff
@@ -76,6 +76,18 @@ type ScheduledWorkflowSpec struct {
 	// +optional
 	Workflow *WorkflowResource `json:"workflow,omitempty"`
 
+	// The pipeline id
+	// +optional
+	PipelineID *string `json:"pipelineID,omitempty"`
+
+	// The pipeline version id
+	// +optional
+	PipelineVersionID *string `json:"pipelineVersionID,omitempty"`
+
+	// The experiment id
+	// +optional
+	ExperimentID *string `json:"experimentID,omitempty"`
```

Comment:
What's your choice for adding pipeline parameters & labels here?
As discussed in https://github.com/kubeflow/pipelines/pull/6207#issuecomment-953256544.

We can have a GVC chat etc if you prefer.

Reply:
Yes, will add it, missed it. Have been jumping around a bit to much between things.

Reply:
No worries, thank you for working on this!